utils/print_mount_paths.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+print_mount_paths() {
+  local mount_path=$1
+  # Print the mounted source path and base mount path, listed in positions 4 and 5 of the resulting string
+  grep "$mount_path" /proc/self/mountinfo | awk '{print $4, $5}'
+}
```

Comment:
What happens if there is more than one match from this grep?

Reply:
Three possible options for dealing with the above (multiple matches for mounted source paths) from most to least complex:

1. Split the result and iterate over it to try and determine which should take precedence
2. Bail if there are multiple results (show some message that the user is doing it wrong)
3. Remove this check as it is really an edge case